src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  return {
+
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, value) {
+      this.result = callback.call(this, value);
 
+      return this;
+    },
+
+    add(value) {
+      return this.result + value;
+    },
+
+    divide(value) {
+      if (value !== 0) {
```

Comment:
delete it we
we can divide at 0 and result will be 0 always

Reply:
Thank you for your review! I absolutely agree - code will be work, but I think, it would be better for continuously using.